tests/test_import_service.py

```
@@ -374,6 +374,80 @@ def test_update_item_sync_override_replaces_existing(self) -> None:
+        item.add_claim.assert_called_once_with("P983", "y_n", qualifiers=[qualifier_claim])
+
+    def test_update_item_sync_override_with_qualifier_replaces_existing(self) -> None:
+        """When overriding an existing value with qualifiers, the old claim is replaced."""
+        old_claim = Mock()
+        old_claim.mainsnak.datavalue = {"value": "y_n"}
+        api, item = self._make_mock_api(existing_claims=[old_claim])
+        qualifier_claim = Mock()
+        api.get_claim.return_value = qualifier_claim
+
+        with patch("services.import_service.MardiClient", return_value=api):
+            payload, ok = import_service.update_item_sync(
```

Ruff (0.15.17)

[warning] Unpacked variable `payload` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)